derp/derp_server.go

```diff
@@ -294,7 +297,10 @@ type Conn interface {
 
 // NewServer returns a new DERP server. It doesn't listen on its own.
 // Connections are given to it via Server.Accept.
-func NewServer(privateKey key.NodePrivate, logf logger.Logf) *Server {
+//
+// clientDataLimit and clientDataBurst configure per-client token-bucket
+// rate/burst limits, in bytes of Wireguard traffic per second.
+func NewServer(privateKey key.NodePrivate, logf logger.Logf, clientdataLimit, clientDataBurst int) *Server {
```

Comment:
rather than extra positional parameters (especially easily-transposable ones of the same type!), I'd prefer a method on Server to set this.